_includes/assets/sass/components/_tags.scss

```
@@ -0,0 +1,6 @@
+.tag {
+  background-color: $tbds-brand-gray-ultra-light;
+  border-radius: 100px;
+  font-size: 0.9rem;
+  padding: $tbds-space-2;
```

```suggestion
  padding: $tbds-space-2 $tbds-space-3;
```
To my eye, just a smidge of x-padding give the labels a little extra breathing room within the tags. What do you think?